shared/js/activeRoomStore.js

```diff
@@ -151,6 +151,8 @@ loop.store.ActiveRoomStore = (function(mozL10n) {
         roomInfoFailure: null,
         // The name of the room.
         roomName: null,
+        // True when sharing screen has been paused.
+        streamPaused: false,
```

Comment:
This needs adding to the list of `_statesToResetOnLeave` - otherwise leaving a room whilst in the pause state, and then re-joining it, does the wrong thing.